README.adoc

```diff
@@ -270,42 +269,45 @@ party service, you should separate your `GET`/`POST` requests from your data con
 // Building and running the application
 // =================================================================================================
 
-[role=command]
-include::{common-includes}/mvnbuild.adoc[]
+== Building and running the application
 
-When the server is running, you can find your service at http://localhost:9080/artists[http://localhost:9080/artists^]
+The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
 
-[role=command]
-include::{common-includes}/mvncompile.adoc[]
+Your application has been built, and a `.war` file has been created in the target directory. Open Liberty was configured and installed into the `target/liberty/wlp` directory.
+
+You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
+
+You can now access the endpoints at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists,
+and at `\http://localhost:9080/artists/total/<artist>` to see a particular artist's total number of albums. 
 
 // =================================================================================================
 // Testing deserialization
 // =================================================================================================
 
 == Testing deserialization
-ConsumingRestTest.java
+ConsumingRestIT.java
 [source, java, linenums, role='code_column hide_tags=comment']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestIT.java[]
 ----
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `ConsumingRestTest` class.#
-`src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestTest.java` 
+#Create the `ConsumingRestIT` class.#
+`src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestIT.java` 
 ----
 
-Maven finds and executes all tests under `it/` and each test method must be marked with the [hotspot=test-1 hotspot=test-2 hotspot=test-3 hotspot=test-4]`@Test` annotation.
+Maven finds and executes all tests under `src/test/java/it/` and each test method must be marked with the [hotspot=test-1 hotspot=test-2 hotspot=test-3 hotspot=test-4]`@Test` annotation.
 
-You can use the [hotspot=BeforeClass]`@BeforeClass` and `@AfterClass` annotations to perform any one time setup and teardown
-tasks before and after all of your tests execute, as well as the [hotspot=Before]`@Before` and [hotspot=After]`@After` annotations
+You can use the [hotspot=BeforeAll]`@BeforeAll` and `@AfterAll` annotations to perform any one time setup and teardown
```

Comment:
@AfterAll hotspot not working.

Reply:
There's no hotspot for `@AfterAll`, because `@AfterAll` is not used in the file. We are just mentioning it in the sentence to say that an `@AfterAll` annotation exists